File: TestLeetCode/Interval/MeetingRoom.cs

 // * Definition of Interval:
 public class Interval {
    public int start, end;
    Interval(int start, int end) {
        this.start = start;
        this.end = end;
    }
}

public class CanAttendMeetingsSolution {
        public bool CanAttendMeetings(List<Interval> intervals) {   
            if (intervals == null || intervals.Count <= 1 ) { return true; }
            intervals.Sort((a, b) => a.start - b.start);
            for (int i = 1; i < intervals.Count; i++)
            {
                if (intervals[i].start < intervals[i - 1].end)
                {
                    return false;
                }
            }
            return true;
        }
}

File: python/interval/meeting_room.py
from python.common import Interval


class MeetingRooms:
    def can_attend_meetings(self, intervals: list[Interval]) -> bool:
        ordered = sorted(intervals, key=lambda item: item.start)
        return all(current.start >= previous.end for previous, current in zip(ordered, ordered[1:]))
